### PR TITLE
Moved donor condition to service

### PR DESCRIPTION
**Pull request by namrataukirde**

### CODE REVIEW

**Comment by namrataukirde** [comment]
Sure @steveyken , Thanks. Will do this change.

File: spec/services/package_condition_mapper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      expect(PackageConditionMapper.to_stockit("Lightly Used")).to eq('M')
+    end
+  end
+
+  describe '.to_condition' do
+    it 'returns donor condtions for stockit mappings' do
+      expect(PackageConditionMapper.to_condition('N')).to eq('New')
+    end
+  end
+end
```
Comment (steveyken) [inline spec/services/package_condition_mapper_spec.rb]: Could you test for empty params too?

File: app/models/package.rb [hunk 2]
```diff
@@ -539,6 +539,10 @@ def dispatched_orders_package
     orders_packages.get_dispatched_records_with_order_id(order_id).first
   end
 
+  def donor_condition_name
+    try(:donor_condition).try(:name_en) || try(:item).try(:donor_condition).try(:name_en)
+  end
```
Comment (steveyken) [inline app/models/package.rb]: This feels very defensively programmed! Do we need all these 'trys'?

    donor_condition.try(:name_en) || item.try(...)

File: spec/services/package_condition_mapper_spec.rb (hunk 1 above)
Comment (namrataukirde) [inline spec/services/package_condition_mapper_spec.rb]: Sure @steveyken

File: app/models/package.rb (hunk 2 above)
Comment (namrataukirde) [inline app/models/package.rb]: We can remove `.try(:name_en)` here.

**Pull request merged**